Hindi demonstration video (#51)
sudiptob2/cf-stats: README.md

# Installation

> ## For step by step screenshot go [here](docs/INSTALLATIONSTEPS.md) 

Reviewer: ```suggestion
### For step-by-step screenshots go [here](docs/INSTALLATIONSTEPS.md) 
### Video Demonstration
  - [Hindi](https://www.youtube.com/watch?v=SMHzbmtHLMo)
```

# Installation

## For step by step screenshot go [here](docs/INSTALLATIONSTEPS.md) 

> ## For video tutorial demonstration (in Hindi) do [visit](https://www.youtube.com/watch?v=SMHzbmtHLMo)

Reviewer: ```suggestion
## Video Demonstration 
- [Hindi](https://www.youtube.com/watch?v=SMHzbmtHLMo)
```

# Installation

## For step by step screenshot go [here](docs/INSTALLATIONSTEPS.md) 

> ## Video Demonstration 

Reviewer: ```suggestion
```

> - [Hindi](https://www.youtube.com/watch?v=SMHzbmtHLMo)

Reviewer: ```suggestion
```
